language-id.el

```diff
@@ -135,7 +135,14 @@
     ("Python" python-mode)
     ("R" ess-r-mode (ess-mode (ess-dialect "R")))
     ("Racket" racket-mode)
-    ("Reason" reason-mode)
+    ("Reason"
+     (reason-mode
+      (language-id--file-name-extension ".re")
+      ))
+    ("ReScript"
+     (reason-mode
+      (language-id--file-name-extension ".res")
+      ))
```

Comment:
Please merge closing parentheses to the previous line of code.

Reply:
Done